.github/workflows/e2e.yaml

```diff
@@ -151,8 +151,16 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      #- name: Extract firmware path
```

Comment:
can we remove this part, if it is commented out?

Reply:
I have uncomment it. We need it in order to extract the `firmwareHostPath` from the configmap instead of hard coding it.